i18n/states.php

```diff
@@ -304,7 +304,24 @@
 		'CN32' => __( 'Xinjiang / 新疆', 'woocommerce' ),
 	),
 	'CZ' => array(),
-	'DE' => array(),
+	'DE' => array(
+		'DE-BW' => __( 'Baden-Württemberg', 'woocommerce' ),
+		'DE-BY' => __( 'Bavaria', 'woocommerce' ),
+		'DE-BE' => __( 'Berlin', 'woocommerce' ),
+		'DE-BB' => __( 'Brandenburg', 'woocommerce' ),
+		'DE-HB' => __( 'Bremen', 'woocommerce' ),
+		'DE-HH' => __( 'Hamburg', 'woocommerce' ),
+		'DE-HE' => __( 'Hesse', 'woocommerce' ),
+		'DE-MV' => __( 'Mecklenburg-Western Pomerania', 'woocommerce' ),
```

Comment:
Noting here that we conform to Unicode CLDR, it seems this should be *Mecklenburg-Vorpommern* [(see here)](https://github.com/unicode-org/cldr/blob/e717766cefb06411b35e4b63953a2b29686d66d2/common/subdivisions/en.xml#L1098):

```suggestion
		'DE-MV' => __( 'Mecklenburg-Vorpommern', 'woocommerce' ),
```